modules/otel-collector-components.adoc

```diff
@@ -557,7 +557,39 @@ config: |
 <2> The default exporter when the attribute value is not present in the table in the next section.
 <3> The table that defines which values are to be routed to which exporters.
 
-You can optionally create an `attribute_source` configuratiion, which defines where to look for the attribute in `from_attribute`. The allowed value is `context` to search the context, which includes the HTTP headers, or `resource` to search the resource attributes.
+You can optionally create an `attribute_source` configuration, which defines where to look for the attribute in `from_attribute`. The allowed value is `context` to search the context, which includes the HTTP headers, or `resource` to search the resource attributes.
+
+[id="cumulativetodelta-processor_{context}"]
+=== Cumulative to Delta Processor
+
```

Comment:
Ideally, the abstract should come before the admonition, though I appreciate that you are trying to avoid stacking the admonitions. This is not a blocker, given the target release window.

I think that you have some information code callouts about the include/exclude parameters and strict/regexp values that you could zoom out on and introduce here. Also, I think it would be useful to explicitly point out that the regexps should be wrapped in quotes.

Reply:
@michaelryanpeter, thank you so much for your review.

I have updated this PR.

Could you please have one more look?

Here's the updated preview: https://75128--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel-configuration-of-otel-collector.html#cumulativetodelta-processor_otel-configuration-of-otel-collector